build-tools/packages/build-tools/src/common/npmPackage.ts

```diff
@@ -150,20 +149,6 @@ export class Package {
 		return this.color(this.name);
 	}
 
-	/**
-	 * The name of the package excluding the scope.
-	 */
-	public get nameUnscoped(): string {
-		return PackageName.getUnscopedName(this.name);
-	}
-
-	/**
-	 * The parsed package scope, including the \@-sign, or an empty string if there is no scope.
-	 */
-	public get scope(): string {
-		return PackageName.getScope(this.name);
-	}
-
```

Comment:
I think these are useful (added in https://github.com/microsoft/FluidFramework/pull/15107). There are lots of times when we have Packages and using these properties is more ergonomic than using the PackageName static all the time. In the places where we do use it, we're working with strings instead of package names. Maybe Package should have a static method instead? Anyway, I think Package is the right place for this to live.

Reply:
This is mostly motivated by reducing unnecessary dependencies.  Those function aren't used by fluid-build, and there are pattern in build-cli that directly call `PackageName` already (even when there is this object), so it should follow that pattern.   

In general, OOP forces to pull in a lot of dependency, and I don't see a lot of value in provide thin wrappers where you can directlly call `PackageName` when needed.

Reply:
Fair enough, if the deps already there. If we really need to expose it from Package in build-cli then we can always subclass or use additionalProps when we create the package.

https://github.com/microsoft/FluidFramework/blob/489cbdedd9c34e9bf1e68da6b8086256ce1f6db8/build-tools/packages/build-tools/src/common/npmPackage.ts#L118-L119